tests/clickhouse-test

```diff
@@ -1617,6 +1615,45 @@ class TestCase:
         # Normalize hostname in stdout file.
         replace_in_file(self.stdout_file, socket.gethostname(), "localhost")
 
+        if args.run_sequential_tests_in_parallel:
```

Comment:
Would it be possible to do this always? I for one have multiple servers and usually run against PORT 49000 so it'd be useful even when not running in parallel:
```
$ cat ~/.bashrc | grep "export CLICKHOUSE"
export CLICKHOUSE_PORT_HTTP=48123
export CLICKHOUSE_PORT_TCP=49000
export CLICKHOUSE_PORT_KEEPER=12181
```

Reply:
Nice idea, changed